x-pack/metricbeat/module/aws/rds/rds.go

```diff
@@ -162,12 +162,16 @@ func getDBInstancesPerRegion(svc rdsiface.ClientAPI) ([]string, map[string]DBDet
 	for _, dbInstance := range output.DBInstances {
 		dbInstanceARNs = append(dbInstanceARNs, *dbInstance.DBInstanceArn)
 		dbDetails := DBDetails{
-			dbArn:              *dbInstance.DBInstanceArn,
-			dbAvailabilityZone: *dbInstance.AvailabilityZone,
-			dbClass:            *dbInstance.DBInstanceClass,
-			dbIdentifier:       *dbInstance.DBInstanceIdentifier,
-			dbStatus:           *dbInstance.DBInstanceStatus,
+			dbArn:        *dbInstance.DBInstanceArn,
+			dbClass:      *dbInstance.DBInstanceClass,
+			dbIdentifier: *dbInstance.DBInstanceIdentifier,
+			dbStatus:     *dbInstance.DBInstanceStatus,
```

Comment:
Are other deferenced values checked at some point? Should we do it?

Reply:
I tested this and these values exist even when the rds just created. So only the AvailabilityZone 😬